src/main/java/com/untamedears/JukeAlert/storage/JukeAlertLogger.java

```diff
@@ -1325,83 +1338,68 @@ public static String createInfoString(SnitchAction entry, boolean censorLocation
             coords = "[*** *** ***]";
         }
         
-        String actionString = "BUG";
+        String actionString = action.toActionString();
         ChatColor actionColor = ChatColor.WHITE;
```

Comment:
Why not continue the pattern, and simply have all this data associated with the enum itself? I'll go back and illustrate what I mean.